docs/blog/datasets_are_books/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en-us">
 
 <head>
-  <meta name="generator" content="Hugo 0.44" />
+  <meta name="generator" content="Hugo 0.30" />
```

Comment:
mind upgrading your copy of Hugo to latest version? that'll make these diffs smaller if we all keep udpating. In the future we should just have CI do this for us, but for now it's easier if we all stay up-to-date